frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,27 @@
+import avatar from '~/assets/img/user-avatar.svg';
+import { Link } from '~/libs/components/link/link.jsx';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  user: unknown;
```

Comment:
And after auth changes implemented we'd change unknown to actual type
```suggestion
  user: unknown | null;
```

Reply:
Can I leave as it is for now? 
Because there is an error from eslint:
type Properties = {
  user: unknown | null;
};
'unknown' overrides all other types in this union type.